goalgorilla/open_social
# #3066658: Add bottom space for private message button

## Problem
The private message button currently relies on the spacing provided by the user's 'function' display. However, if the user does not fill this in on their profile the spacing for the private message button is incorrect.

## Solution
Provide the private message button with the appropriate spacing in the socialbase module so that it looks the same whether the function is filled in on the profile or not.

## Issue tracker
https://www.drupal.org/project/social/issues/3066658

## How to test
- Should enable 'Social Private Message' module
- Go to profile page any more user and add/deleted text on 'Function and organization' on edit  page

## Release notes
Private message button has its own spacing
